# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../afio/install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- afio-2.5/Makefile.vanilla	2003-12-21 11:23:51.000000000 +0100
+++ afio-2.5/Makefile	2005-08-09 17:58:47.000000000 +0200
@@ -88,8 +88,8 @@
 	cd regtest; /bin/sh regtest.clean
 
 install: afio
-	cp afio /usr/local/bin
-	cp afio.1 /usr/share/man/man1
+	cp afio /usr/bin
+	cp afio.1 /usr/man/man1
 
 # generate default list of -E extensions from manpage
 # note: on sun, I had to change awk command below to nawk or gawk
